CHANGES.md

```diff
@@ -3,10 +3,13 @@ Change Log
 
 ### 1.40 - 2017-12-01
 
+* Deprecated
+  * The `text`, `imageUrl` and `link` parameters for `Credit` have been deprecated and will be removed in Cesium 1.41.  Use `options.text`, `options.imageUrl` and `options.link` instead.
```

Comment:
Just a reminder to submit an issue for this.

Reply:
@hpinkos did I miss this new issue?  No issues are labeled remove in 1.41 - https://github.com/AnalyticalGraphicsInc/cesium/labels/remove%20in%201.41

Reply:
Thanks for the reminder @pjcozzi!